### ISSUE TITLE
User remains on the new “Connection information” screen after turning the VPN OFF

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#5377]
**Important details:** 

* “Connection information” screen is no longer displayed if pressing the “Home” button from the navigation bar;

* This is also reproducible on mobile, if turning the VPN OFF from the notification bar on Android, or if turning it OFF from the device settings - VPN menu, on iOS; 

**Affected versions:**

* 2.13.0 (2.202301022206);

**Tested Platforms:**

* All;

**Prerequisites:**

* Have Mozilla VPN installed and opened;

* Have a Fx account with an active subscription;

* Be signed in;

* Have the VPN in ON state;

**Steps to reproduce:**

1. Click on the IP info button, placed on the top right side of the activation screen;
1. Turn OFF the VPN from the system tray bar;
1. Observe the behavior;

**Expected result:**

* “Connection information” screen is no longer displayed, and instead the “VPN is off” screen is displayed.

**Actual result:**

* “Connection information” screen is still displayed, and the “x” button disappears. The user remains on the same screen. 

**Notes:**

* See the attached video;

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-3613)
┆Reporter: Bianca Hidecuti

**Comment by data-sync-user** [comment]
➤ Bianca Hidecuti commented:

Lesley Norton, could you please add the fix version? Thank you.

**Comment by data-sync-user** [comment]
➤ Bianca Hidecuti commented:

Verified this as fixed on the latest 2.13 builds (2.13.0 (2.202301161600)) from the release branch, across platforms.

“Connection information” screen is no longer displayed when turning the VPN off from the system tray menu / device settings on mobile.

Attaching postfix video. 

I also added the 2.13 fix version, as the fix is merged into the release branch.

!vpn-3613.mov|width=1338,height=1342!
